### DISCUSSION TITLE
Zavala 2.6

### DISCUSSION THREAD

**Discussion by vincode-io** [vincode-io/Zavala#243] [General]
I've been testing Zavala 2.6 which hopefully will be released soon. There are currently no outstanding issues that I am aware of. We just need to do more testing.

In this release I've added Typewriter Scrolling and Row Level Linking. The Settings categories have also been revised to be more consistent. There are also a couple of bug fixes in there as usual.

One change that was long overdue, might be controversial. Currently when exporting Markdown or OPML and the Outline contained a link to another Outline, we translated the link from a Zavala Deep Link to a File Relative Link. The assumption being that if you were exporting one Outline, you were probably also exporting the linked one for use in another editor and didn't want a link back to Zavala.

In hindsight that was a bad design decision which has been reverted. If you still need to export multiple Outlines and want the File Relative Link to be used, the Export Shortcut has been updated to allow you to do this. I feel like this is the best compromise to keep backwards functionality without overly complicating the Zavala UI.

**Comment by vincode-io** [comment]
2.6 is now live! Let me know what you think about this release in the comments below.

**Comment by jsismail** [comment]
Row linking is very very nice.  Thank you!
